### PR TITLE
Updated Food Channel (Domino's) guide to be up to date

### PR DESCRIPTION
**Pull request by PablosCorner**
Updated the Food Channel (Domino's) guide from WiiLink to be up to date.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -25,6 +25,7 @@ GEM
       ruby2_keywords (>= 0.0.4)
     faraday-net_http (2.0.3)
     ffi (1.15.5)
+    ffi (1.15.5-x64-mingw-ucrt)
```
Comment (Peter0x44) [inline Gemfile.lock:28]: Just wondering, what are these changes for?

Comment (PablosCorner) [reply Gemfile.lock:28]: I was using Ruby with MinGW on Windows, so it auto added that.

File: _pages/en_US/wiilink-demae-dominos.md
Comment (Snare-Hawk) [inline _pages/en_US/wiilink-demae-dominos.md:1]: this looks great! only thing is that all instances of "Food Channel" should be changed to "Food Delivery Channel" as that is what sketch said it should be referred to as
~~i'd try and do it myself if i could right now, but i can't lol~~

Comment (PablosCorner) [reply _pages/en_US/wiilink-demae-dominos.md:1]: Ahh, I was showing my progress with writing the guide to Sketch and he seemed to think it was good as-is. So, I'm guessing he either changed his mind on the naming, or at least was okay with me calling it just "Food Channel". You're welcome to ask him if that's the case, at least.

**Comment by PablosCorner** [comment]
@Snare-Hawk I made the changes

**Comment by PablosCorner** [comment]
For some odd reason the new SPD images aren't showing up on the live site. 🤔

**Pull request merged**